egs/babel/s5d/local/prepare_unicode_lexicon.py

```diff
@@ -25,173 +24,170 @@
 import os
 import argparse
 
-SKIP = ("", "''", "<", ">", "#")
-
 
-# Extract a sorted set of distinct unicode graphemes from the lexicon
-def extract_graphemes(table):
+# Extract a sorted set of distinct phonemes from the lexicon
```

Comment:
when I'm looking at the changes, perhaps important part for a person reading those script would be to understand the distinction between graphemes and phonemes -- can you please document that? Another thing to consider if we want the acoustic units phonemes (but I think that would be ok, if it's documented what that means).